package.json

```diff
@@ -3,6 +3,7 @@
   "version": "2.0.5",
   "description": "React component for truncating multi-line spans and adding an ellipsis",
   "main": "lib/Truncate.js",
+  "module": "src/Truncate.js",
   "files": [
     "lib"
```

Comment:
You need to include `"src"` here as well, otherwise it won't be included in the npm package